CodingGuidelines.md

```diff
@@ -212,43 +250,43 @@ public class MyClass
 
  ### Do:
 
- ```
+ MyStruct.cs
+ ```c#
  // Public Struct / Enum definitions for use in your class.  Try to make them generic for reuse.
-MyStruct.cs
 public struct MyStruct
 {
     public string Var1;
     public string Var2;
 }
 ```
 
-```
 MyEnumType.cs
+```c#
 public enum MuEnumType
 {
     Value1,
     Value2 // <- note, no "," on last value to denote end of list.
 }
 ```
 
-```
 MyClass.cs
+```c#
 public class MyClass
 {
     private MyStruct myStructreference;
     private MyEnumType myEnumReference;
 }
  ```
 
-## Initilize Enums.
+## Initialize Enums.
 
-To ensure all Enum's are initialized correctly starting at 0, .NET gives you a tidy shortcut to automatically initilize the enum by just adding the first (starter) value.
+To ensure all Enum's are initialized correctly starting at 0, .NET gives you a tidy shortcut to automatically initialize the enum by just adding the first (starter) value.
```

Comment:
Other than readability, what is the reason to require this? .net automatically initializes the first emum value to 0 unless otherwise specified.

Reply:
because it's good practice, this was one of the first guidelines added when we started the document.  To ensure consitancy across the solution. (also recommended in MSDN I believe from the discussions we had back then.)